viewerutil/sign_windows.py

```diff
@@ -1,42 +1,25 @@
-#!/usr/bin/env python3
-"""\
-@file   sign.py
-@author Nat Goodspeed
-@date   2023-09-14
-@brief  Sign the designated executable using Microsoft AzureSignTool.
-
-$LicenseInfo:firstyear=2023&license=viewerlgpl$
-Copyright (c) 2023, Linden Research, Inc.
-$/LicenseInfo$
```

Comment:
These kinds of machine readable license headers [are not needed anymore](https://wiki.secondlife.com/wiki/Coding_standard#MIT_Licensed_Code), a LICENSE in root suffices.

Reply:
So noted, thanks.